datasrc/compile.py

```diff
@@ -1,4 +1,4 @@
-import os, imp, sys
+import os, importlib, sys
```

Comment:
```suggestion
import os
import sys
```
The `imp` import can probably be entirely removed if replacing it with `importlib` did not break anything.